tests/test-cli.sh

```diff
@@ -21,6 +21,7 @@ ${OASIS_REGISTRY} verify
 
 # Create new entity metadata.
 ${OASIS_REGISTRY} entity update \
+    --assume_yes \
```

Comment:
indentation doesn't seem right here (and bellow)

Reply:
Nice catch, this was due to mixture of tabs and spaces.

Reply:
I've added Shell code formatting and verification via the [shfmt tool](https://github.com/mvdan/sh#shfmt)